host/bcm283x/dma.go

```diff
@@ -642,6 +643,54 @@ func runIO(pCB pmem.Mem, liteOk bool) error {
 	return ch.wait()
 }
 
+func allocateCB(size int) ([]controlBlock, *videocore.Mem, error) {
+	buf, err := videocore.Alloc((size + 0xFFF) &^ 0xFFF)
+	if err != nil {
+		return nil, nil, err
+	}
+	// TODO(simokawa): call buf.Close()
+	var cb []controlBlock
+	if err := buf.AsPOD(&cb); err != nil {
+		buf.Close()
+		return nil, nil, err
+	}
+	return cb, buf, nil
+}
+
+func startPWMbyDMA(p *Pin, rng, data uint32) (*dmaChannel, *videocore.Mem, error) {
+	cb, buf, err := allocateCB(4096)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := buf.Uint32()
+	cb_bytes := uint32(32)
```

Comment:
"_" in symbol names is not idiomatic in Go except rare circonstances.

Reply:
Renamed.